Update branch policy (#5363)
plone/volto: packages/registry/CHANGELOG.md

# @plone/registry Release Notes

> <!-- You should *NOT* be adding new change log entries to this file.

Reviewer: ```suggestion
<!-- Do *NOT* add new change log entries to this file.
```

     You should create a file in the news directory instead.
     For helpful instructions, please see:

>      https://6.docs.plone.org/volto/developer-guidelines/contributing.html#create-a-pull-request

Reviewer: This has moved.

```suggestion
     https://6.docs.plone.org/contributing/index.html#contributing-change-log-label
```
